manifests/gcp_marketplace/schema.yaml

```diff
@@ -122,8 +122,30 @@ properties:
       Using CloudSQL and GCS provides better reliability and performance,
       as well as features such as data backup and usage monitoring.
       This is the recommended option especially for production scenarios.
-      If false, the data will be stored in GCE Persistent Disk.
+
+      If yes, you must specify GCS bucket and CloudSQL instance below. Note, to
```

Comment:
@joeliedtke It will be really helpful if you can review these documentation changes.